apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/FinalDecisionDialog.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { Meta, StoryFn } from "@storybook/react";
+
+import { fakePoolCandidates, fakeSkills } from "@gc-digital-talent/fake-data";
+import { OverlayOrDialogDecorator } from "@gc-digital-talent/storybook-helpers";
+import {
+  AssessmentDecision,
+  AssessmentResult,
+  AssessmentResultType,
+} from "@gc-digital-talent/graphql";
+
+import FinalDecisionDialog from "./FinalDecisionDialog";
+
+export default {
+  component: FinalDecisionDialog,
+  title: "Components/Final decision dialog",
+  decorators: [OverlayOrDialogDecorator],
```

Comment:
I thought this makes the dialog open by default in storybook but doesn't seem to be the case

Reply:
You will need to add a `defaultOpen` prop and pass that in with the args. Checkout the [FilterDialog](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/apps/web/src/components/FilterDialog/FilterDialog.stories.tsx) for an example of this.

The decorator just expands the canvas of the story so we can show the dialog. Otherwise, it is scoped to just the button and it won't appear.

Reply:
Ah okay thanks, yeah I needed the prop to be added in

https://github.com/GCTC-NTGC/gc-digital-talent/pull/8978/commits/12d8b82c7b3c5899ec1efe7c872af8914ad59919